Index.html

```diff
@@ -0,0 +1,153 @@
+               </p>
+
+               <p class="eventDate">2022.12.10(SAT) ~ 11(SUN)</p>
+               <span class="venue">At Congrate hall Bamenda Cameroon</span>
+        </div>
+      </section>
+
+      <section class="shedule-section">
+        <div class="text-time">
+          <h2 class="text-header">Main Program</h2>
+          <hr class="line">
+        </div>
+        
+        <div class="shedule-container">
+          <article class="shedule-cards">
+            <img class="image" src="Images/Musician/dj.png"  alt="Djs-icon ">
+            <h3>Openning with JD</h3>
+            <p class="text1">Listen to theamazing talented DJs in Bemanda. Give you what they do best in their career</p>
+          </article>
+
+          <article class="shedule-cards">
+            <img class="image" src="Images/Musician/artist.png" alt="First-artist-icon">
+            <h3>1<sup>st</sup> Artist</h3>
+            <p>The fist artist will take over the stage after the Djs have finish performing their talent</p>
+          </article>
+
+          <article class="shedule-cards">
+            <img class="image" src="Images/logo_image/2artise-removebg-preview.png" alt="Second-artist-icon">
+            <h3>2<sup>nd</sup> Artist</h3>
+            <p>The second artist will take over the stage after the 1st artist have finish performing their talent</p>
+          </article>
+
+          <article class="shedule-cards">
+            <img class="image" src="Images/logo_image/images-removebg-preview.png" alt="Third-artist-icon">
+            <h3>3<sup>th</sup> Artist</h3>
+            <p>The third artist will take over the stage after the 2nd artist  have finish performing their talent</p>
+          </article>
+
+          <article  class="shedule-cards">
+            <img class="image" src="Images/logo_image/Mic_inside_Pin_Street_Podcast_logo__2_-removebg-preview.png" alt="Main-artist-icon">
+            <h3>Main Artist</h3>
+            <p>The main artist of the day will close the program and trust US he will blow your mind</p>
+          </article>
+        </div>
+        <div class="text-time" >
+          <a class="text-header program" href="#">SEE THE WHOLE PROGRAM</a>
+        </div>
+        <div class="button-container">
+        <button class="ticket-button" type="submit">
+             Grab a ticket today
+        </button>
+        </div>
+      </section>
+
+      <section class="artist-section setoff">
+        <div class="text-time">
+        <h3 class="text-header2">Artists</h3>
+        <hr class="line2">
+        </div>
+
+        <div class="artist-container">
+       
+        </div>
+
+      </section>
+      <div class="seemore">
+        <span id="span">see more artist &nabla;</span>
+      </div>
+
+      <section class="spnosor-section">
+        <div class="text-time">
+             <h3  class="text-header">Spnosors</h3>
```

Comment:
- [ ] As the previous code reviewer mentioned that you should write it `Sponsors` not `Spnosors`.